apps/odk_publish/admin.py

```diff
@@ -54,12 +55,18 @@ class ProjectTemplateVariableInline(admin.TabularInline):
 
 @admin.register(Project)
 class ProjectAdmin(admin.ModelAdmin):
-    list_display = ("name", "central_id", "central_server", "organization")
+    list_display = ("name", "central_id", "central_server", "organization", "app_language")
     search_fields = ("name", "central_id")
     list_filter = ("central_server",)
     filter_horizontal = ("template_variables",)
     inlines = (ProjectAttachmentInline, ProjectTemplateVariableInline)
 
+    def save_model(self, request, obj, form, change):
+        super().save_model(request, obj, form, change)
+        # Regenerate app user QR codes if the app_language has changed
+        if change and "app_language" in form.changed_data:
```

Comment:
I realized when reviewing this that `central_id.central_id` and `project.name` also impact the QR and could be checked here: https://github.com/caktus/odk-publish/blob/main/apps/odk_publish/etl/load.py#L146-L151